src/vector_store/composite.py

```
@@ -0,0 +1,261 @@
+        try:
+            results = await self.primary.query(
+                namespace,
+                embedding,
+                top_k=top_k,
+                filters=filters,
+                max_distance=max_distance,
+            )
+            logger.debug(
+                f"Primary query returned {len(results)} results from namespace {namespace}"
+            )
+            return results
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)